doc/nrf/known_issues.rst

```diff
@@ -157,6 +157,16 @@ Other issues
 
 .. rst-class:: v2-0-0
 
+NCSDK-15909: Crypto samples fail to build because of flash overflow with Zephyr SDK 0.14.2 on nRF5340
```

Comment:
There could be other samples other than the crypto ones that fail to build (secure_peripheral_partition for example).
Also, the specific to nrf53 is too narrow in my opinion. It is likely very few code-lines of difference between the two platforms, so probably wouldn't take much to tip nrf9160 over. In addition turning a feature off for the nrf53 could also make it compile again.

Suggest:
TF-M may fail to build because of flash overflow with Zephyr SDK 0.14.2 when TFM_PROFILE_TYPE_NOT_SET=y